app/components/select-list.js

```diff
@@ -0,0 +1,84 @@
+import Ember from "ember";
+import ValidatableInput from 'ember-cli-html5-validation/mixins/validatable-input';
+
+export default Ember.Component.extend(ValidatableInput, {
+  content: null,
+  selectedValue: null,
+  pkg: null,
+  index: null,
+  flag: true,
+
+  didRender() {
+    this._super(...arguments);
+    if ( this.pkg != null && this.pkg.packageType != null && this.flag === true){
```

Comment:
You always need to use Em.get and Em.set for getting and setting properties of an Ember.Object. It's the basic rule. Without it you may find variety of bugs in observers/rendering and other places.